### ISSUE TITLE
Packages Permitidos 

### ISSUE THREAD

**Issue by nateare** [IIC2513-2018-1/syllabus#15]
Se puede usar el package _bluebird_ en la app? En la wiki solo sale que no se pueden usar CSS Frameworks.

**Comment by gpvidal** [comment]
Sí, no habría problema. Sin embargo, te recomiendo leer aquí su documentación por si quieres utilizar alguna funcionalidad de Bluebird que ya está en las Promesas:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise

Saludos!

**Comment by nateare** [comment]
Yap, gracias!

**Comment by nateare** [comment]
Quería saber si se puede usar `leaflet` o `mapbox`, también podría ser `@google/maps`?

**Comment by gpvidal** [comment]
Sí, no hay problema con ocupar una librería para mapas.

Saludos!

**Comment by nateare** [comment]
Ok, gracias
